src/main/java/de/luhmer/owncloud/accountimporter/adapter/AccountImporterAdapter.java

```diff
@@ -43,28 +43,26 @@ public View getView(final int position, View view, ViewGroup parent) {
             TextView text2 = (TextView) view.findViewById(R.id.text2);
             CheckBox cbChecked = (CheckBox) view.findViewById(R.id.checkbox);
             holder = new ViewHolder(text1, text2, cbChecked);
-
             view.setTag(holder);
         }
 
-        holder.text1.setText(getItem(position).type);
-        holder.text2.setText(getItem(position).url);
-        holder.cbChecked.setChecked(getItem(position).checked);
+        String username = getItem(position).name.split("@")[0];
+        String server   = getItem(position).name.split("@")[1];
```

Comment:
Newest accounts should have userId@server, instead of loginname@server.
UserId is a unique id per server
Previously we used the loginname, which also can be an email address.

To also have this working with legacy accounts, this should be splitted be last "@":
tobias@mail.com@nextcloud.cloud.com -> tobias@mail.com (username) & nextcloud.cloud.com (server)

Reply:
Preview of code: https://repl.it/repls/KookyTriangularInterchangeability

Better ideas on how to implement this? Unfortunately Java does not offer something like "split last"..